graphql/queries/Orders.graphql

```diff
@@ -0,0 +1,16 @@
+query Orders {
```

Comment:
I would instead use here different name for this query, for example `RecentOrders`. There are probably already a few classes called `Orders` generated by codegen, which might confuse first-time GQL users. Giving a meaningful name for a query is also a good practice.